pandas/tests/extension/test_common.py

```diff
@@ -79,3 +133,20 @@ def test_astype_no_copy():
 def test_is_extension_array_dtype(dtype):
     assert isinstance(dtype, dtypes.ExtensionDtype)
     assert is_extension_array_dtype(dtype)
+
+
+@pytest.mark.parametrize("na_value", [np.nan, pd.NA, None])
+def test_empty_series_construction(na_value):
```

Comment:
why doing this here instead of in extensions.base so we get the test for every dtype?

Reply:
I don't have a good reason other than I needed to define an Extension Array and Dtype and this module had ones that I could extend.  It's one of the few modules which seemed to be testing against a defined Extension Array/Dtype.   From [these docs](https://pandas.pydata.org/docs/development/extending.html#testing-extension-arrays) I assumed that extensions.base existed to provide tests for users to apply to their own custom Dtypes, rather than existing to test pandas code.   Are you suggesting I use the Extension Array and Dtype I've defined, populate the fixtures found [here](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/extension/conftest.py) and have the entire extension test suite run over this Extension Array/Dtype?

Reply:
@jbrockmendel should I just create a separate module for the Dtype and ExtensionArray, in which the necessary pytest fixtures are also defined, and run the entire EA test suite against it?

Reply:
my thought was that test_empty_series_construction would be a test in tests.extension.base.constructors.BaseConstructorsTests.  But now that i read more carefully i guess none of the existing EAs in the test suite have the characteristics relevant to this particular bug?

Reply:
hi @jbrockmendel that is correct